crates/burn-wgpu/src/lib.rs

```diff
@@ -22,7 +22,7 @@ pub type SpirV = cubecl::wgpu::spirv::VkSpirvCompiler;
 #[cfg(feature = "spirv")]
 type Compiler = SpirV;
 #[cfg(feature = "spirv")]
-type Byte = u8;
+type Bool = u8;
```

Comment:
Is it ok?

Reply:
Unrelated to this PR, but if you run the checks it fails for spirv

```
error[E0412]: cannot find type `Bool` in this scope
  --> crates/burn-wgpu/src/lib.rs:63:37
   |
63 | pub type Wgpu<F = f32, I = i32, B = Bool, C = Compiler> =
   |                                     ^^^^
   |
help: a builtin type with a similar name exists
   |
63 | pub type Wgpu<F = f32, I = i32, B = bool, C = Compiler> =
   |                                     ~~~~
help: perhaps you intended to use this type
   |
63 | pub type Wgpu<F = f32, I = i32, B = bool, C = Compiler> =
   |                                     ~~~~
help: consider importing this struct
   |
12 + use burn_tensor::Bool;
   |

For more information about this error, try `rustc --explain E0412`.
```

so I fixed it here. I think it was a leftover from the byte type rename.